packages/components/select/stories/select.stories.js

```diff
@@ -33,6 +33,7 @@ export const Default = () => {
 	const disabled = boolean('disabled', false);
 	const opened = boolean('opened', false);
 	const multiselect = boolean('multiselect', false);
+	const noApplyButton = boolean('multiselect', false);
```

Comment:
nit: please, rename from `multiselect` to `no apply button` or similar.

Reply:
good catch